source/common/upstream/health_discovery_service.h

```diff
@@ -97,6 +98,7 @@ class HdsCluster : public Cluster, Logger::Loggable<Logger::Id::upstream> {
   std::vector<Upstream::HealthCheckerSharedPtr> health_checkers_;
   HealthCheckerMap health_checkers_map_;
   TimeSource& time_source_;
+  UnitFloat drop_overload_{0};
```

Comment:
Do we need to parse the HDS cluster config and store the drop_overload config here?